lib/bap/bap.mli

```diff
@@ -5218,6 +5218,11 @@ module Std : sig
     (** symbol  *)
     type symbol [@@deriving bin_io, compare, sexp]
 
+    (** code region  *)
```

Comment:
we don't need this in the `Image` interface at all. Please remove `{code,data}_region` data types and functions.